src/Files.App/ViewModels/UserControls/InfoPaneViewModel.cs

```diff
@@ -117,17 +117,24 @@ public InfoPaneViewModel()
 			IsEnabled = infoPaneSettingsService.IsEnabled;
 		}
 
-		private void ContentPageContext_PropertyChanged(object? sender, PropertyChangedEventArgs e)
+		private async void ContentPageContext_PropertyChanged(object? sender, PropertyChangedEventArgs e)
 		{
 			switch (e.PropertyName)
 			{
 				case nameof(IContentPageContext.Folder):
 				case nameof(IContentPageContext.SelectedItem):
 
+					ListedItem selectedItem = null;
```

Comment:
Isn’t this making a warning?
Make sure it is not nullable
= null!; works better

Reply:
Or make it nullable and add null check.

Reply:
`SelectedItem` property is not nullable either to begin with, so they should be considered together.
Also, `selectedItem` is also the name of the backing field of `SelectedItem`, so it is better to change the name.

Reply:
Yes this creates a warning, as it was there before I ignored it (line 130 `SelectedItem = null;`)

I have now renamed the variable to `tempSelectedItem` to avoid conflict with the backing field and made both nullable